tests/e2e/workloads/bdi/bdi_base_class.py

```diff
@@ -0,0 +1,239 @@
+import pytest
+from datetime import datetime
+import logging
+import tempfile
+from ocs_ci.framework.testlib import E2ETest, workloads
+from ocs_ci.ocs import ocp
+from ocs_ci.utility import templating
+from ocs_ci.framework import config
+
+from ocs_ci.helpers.bdi_helpers import (
+    init_configure_workload_yaml,
+    init_data_load_yaml,
+    init_run_workload_yaml,
+    install_db2u,
+    clean_temp_files,
+)
+
+log = logging.getLogger(__name__)
+
+
+@workloads
+class TestBdiWorkloadBaseClass(E2ETest):
+    chart_dir = "/stable/ibm-db2warehouse/ibm_cloud_pak/pak_extensions"
+    chart_git_url = "https://github.com/IBM/charts"
+    tiller_namespace = "tiller"
+    helm_tar_file = "helm-v2.9.0-linux-amd64.tar.gz"
```

Comment:
I think it would be better to save these different parameters in a config file as yaml or something. This will make it easier when a change is needed to be done

Reply:
Done